runtime/vm/LayoutFFITypeHelpers.hpp

```diff
@@ -42,13 +42,14 @@ class LayoutFFITypeHelpers
 	/* Function members */
 public:
 	LayoutFFITypeHelpers(J9VMThread *currentThread)
-			: _currentThread(currentThread)
-			, _vm(_currentThread->javaVM)
-	{ };
+					: _currentThread(currentThread)
+					, _vm(currentThread->javaVM)
+	{
+	}
 
 	/**
 	 * @brief Convert the preceding integer of a layout string from string to UDATA,
-	 * and put layout at the position after the integer
+	 * and put layout at the position after the integer.
 	 *
 	 * @param layout[in] A pointer to a c string describing the types of the struct elements. For example:
```

Comment:
Please consistently refer to "C string" (uppercase C).

Reply:
Fixed.